tabular/tests/unittests/models/test_image_predictor.py

```diff
@@ -0,0 +1,16 @@
+
+from autogluon.core.features.feature_metadata import FeatureMetadata
+from autogluon.tabular import TabularPredictor
+from autogluon.vision import ImageDataset
+
+
+def test_image_predictor(fit_helper):
+    train_data, _, test_data = ImageDataset.from_folders('https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip')
+    feature_metadata = FeatureMetadata.from_df(train_data).add_special_types({'image': ['image_url']})
+    predictor = TabularPredictor(label='label').fit(
+        train_data=train_data,
+        hyperparameters={'AG_IMAGE_NN': {'epochs': 2}},
+        feature_metadata=feature_metadata
+    )
```

Comment:
maybe also test setting `predictor=None` and reloading it before inference

Reply:
That is tested elsewhere, since we know predictor doesn't persist models in memory be default.